### ISSUE TITLE
Multiple API keys?

### ISSUE THREAD

**Issue by nedbaker12** [rudranag/Truecaller-Smsbomber_telegram_bot#10]
are you able to feature add multiple True caller API keys so if one doesnt work other work?

**Comment by rudranag** [comment]
Adding the features is okay but the problem is getting those API keys.

**Comment by nedbaker12** [comment]
does the bearer token expire everyday?

**Comment by rudranag** [comment]
Actually that token expired long ago. But I will soon update a token

**Comment by nedbaker12** [comment]
no i m asking when you generate a bearer token from truecaller. will it expire every day?

**Comment by rudranag** [comment]
No. But if we use the API too many times in a specific interval ( too many requests in a minute) . it gets suspended for 24 hours.
